app/js/services.js

```diff
@@ -0,0 +1,57 @@
+floripaApp.factory('searchService', function () {
+    return {};
+});
+
+floripaApp.factory('FindRoutesByStopName', ['$resource', '$http',
+
+  function ($resource, $http) {
+
+        $http.defaults.headers.common['Authorization'] = authString;
+
+        return $resource(api_url + 'findRoutesByStopName/run/', {}, {
+            save: {
+                method: 'POST',
+                headers: {
+                    'X-AppGlu-Environment': 'staging',
+                    'Content-type': 'application/json'
+
+                }
+            }
+        });
+}]);
+
+floripaApp.factory('FindStopsByRouteId', ['$resource', '$http',
+
+  function ($resource, $http) {
+
+        $http.defaults.headers.common['Authorization'] = authString;
+
+        return $resource(api_url + 'findStopsByRouteId/run/', {}, {
+            save: {
+                method: 'POST',
+                headers: {
+                    'X-AppGlu-Environment': 'staging',
+                    'Content-type': 'application/json'
+
+                }
+            }
+        });
+}]);
+
+floripaApp.factory('FindDeparturesByRouteId', ['$resource', '$http',
+
+  function ($resource, $http) {
```

Comment:
I would create a single service for the calls, called timetableSvc. Ir order to call the service, first you have to set the action. The actions are pre-defined inside of the service with the 3 strings, and you just set it with something like timetable.action = timetable.actions.FindStopsByRouteId.